nixos/modules/phoebus/alarm-server.nix

```diff
@@ -76,18 +76,6 @@ in {
             defaultText = lib.literalExpression ''"localhost:''${toString config.services.apache-kafka.port}"'';
           };
 
-          "org.phoebus.applications.alarm/config_name" = lib.mkOption {
```

Comment:
Left as commented code, because it will be the option used in the future, instead of `config_names`